test-site/src/index.ts

```diff
@@ -1,90 +1,8 @@
-import {CtaClick, provideAnalytics, providePagesAnalytics, provideChatAnalytics} from '@yext/analytics';
-import { CookieManager, provideConversionTrackingAnalytics } from '../../src';
+import analytics from '../../src/index';
```

Comment:
I think the import from `'@yext/analytics'` was intentional as this is meant to simulate how a user would interact with the SDK.

Is this import you used functionally the same?